src/levanter/infra/ray_tpu.py

```diff
@@ -62,21 +63,28 @@ class TpuRunError(_TpuRunResult):
     error: Exception
 
 
-def run_on_pod(remote_fn: RemoteFunction | Callable, tpu_type: str) -> ray.ObjectRef:
+def run_on_pod(docker_cmd: DockerRunCommand, name: str, tpu_type: str) -> ray.ObjectRef:
```

Comment:
hrm, we use the non-docker version in Marin so we do actually want the non-docker version to work.

Do we need this change?

Reply:
single slice doesn't need this change. I only changed it to make it consistent with the multislice. I don't think there's a good way to undo this change for multislice tho.